src/ripple/net/impl/HTTPClient.cpp

```diff
@@ -411,27 +411,22 @@ class HTTPClientImp
             mBody = smMatch[1];
 
         if (boost::regex_match (strHeader, smMatch, reSize))
-        {
-            int size = beast::lexicalCastThrow <int> (std::string(smMatch[1]));
-
-            if (size < mResponseMax)
-                mResponseMax = size;
-        }
```

Comment:
No upper bound anymore? Should we, maybe, have a `boost::clamp` here to set sensible lower and upper limits?